File: C Codes/Cpp/Basics/branching.cpp
#include<iostream>
using namespace std;
int main()
{
    int a=25,b=80,c=160;
    if(a>b) {
        if(a>c)  {
            cout<<"A is greater: "<<a<<endl;
        }
        else{
            cout<<"C is greater: "<<c<<endl;
        }
    }
    else if(b>c) {
        cout<<"B is Greater:"<<b<<endl;
    }
    else  {
        cout<<"C is Greater: "<<c<<endl;
    }
}

File: Java/Java Code/Innovative/src/Basics/Branching.java
	package Basics;

import java.util.Scanner;

public class Branching {

	public static void main(String[] args) 
	{
		Scanner d = new Scanner(System.in);
		System.out.println("Enter 3 values: ");
		int a= d.nextInt(); //100
		int b = d.nextInt();//50
		int c = d.nextInt();//200
		
		if(a>b) //100>50
		{
			if(a>c) //100>200
				System.out.println("A is Greater: "+a);
			else
				System.out.println("C is Greater: "+c);
		}
		else if(b>c)//50>200
			System.out.println("B is Greater: "+b);
		else
			System.out.

					println("C is Greater: "+c);
		
	}
}
